src/client.js

```diff
@@ -307,38 +328,65 @@ class Admin {
     this.client = new StitchClient('', {baseUrl});
   }
 
-  _do(url, method, options) {
-    return this.client._do(url, method, options)
-      .then(response => response.json());
-  }
-
-  _get(url, queryParams) {
-    return this._do(url, 'GET', {queryParams});
-  }
-
-  _put(url, options) {
-    return this._do(url, 'PUT', options);
+  get _v2() {
```

Comment:
[q] What's the intent of making these getters? Sugar?

Reply:
yeah, sugar. this method is not consumed outside the Admin class.